docs/cloud/reference/warehouses.md

```diff
@@ -122,8 +122,6 @@ Once compute-compute is enabled for a service (at least one secondary service wa
 
 ## Limitations {#limitations}
 
-Because this compute-compute separation is currently in private preview, there are some limitations to using this feature. Most of these limitations will be removed once the feature is released to GA (general availability):
-
 1. **Primary service should always be up and should not be idled (limitation will be removed some time after GA).** During the private preview and some time after GA, the primary service (usually the existing service that you want to extend by adding other services) will be always up and will have the idling setting disabled. You will not be able to stop or idle the primary service if there is at least one secondary service. Once all secondary services are removed, you can stop or idle the original service again.
```

Comment:
@kapustor do these limitations still apply, and if not which can I remove?

Reply:
@Blargian yes, all limitations are still in place (unfortunately). We are waiting for the shared catalog roll out to remove them.